platform/ios/CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
-## 3.7.3
+## 3.7.4
+
+* Added the `MGLTileSourceOptionTileCoordinateBounds` option to create an `MGLTileSource` that only supplies tiles within a specific geographic bounding box. ([#11141](https://github.com/mapbox/mapbox-gl-native/pull/11141))
```

Comment:
This blurb (and the corresponding macOS blurb) will need to be removed from the “master” section once v3.7.4 is released and release-agua is merged back to master.